docs/feel-unary-tests.md

```diff
@@ -28,7 +31,7 @@ Compare the input value to `x`.
   * year-month-duration
   * day-time-duration 
 
-### Inveral
+### Interval
 
 Test if the input value is within the interval `x` and `y`.
 
```

Comment:
Next line: 
An interval can be open or close**d**. If the interval is open **then** the value _at the boundary_ is not included.

Reply:
Good catch!